modules/template_files/main.tf

```
@@ -14,6 +14,10 @@
  * limitations under the License.
  */
 
+locals {
+  startup_script_content = var.append_to_startup_script == null ? data.template_file.startup_script.rendered : "${data.template_file.startup_script.rendered}${data.local_file.additional_startup_script[0].content}"
```

We should include newline characters to ensure that the two scripts do not collide.

```suggestion
  startup_script_content = var.append_to_startup_script == null ? data.template_file.startup_script.rendered : "${data.template_file.startup_script.rendered}\n\n${data.local_file.additional_startup_script[0].content}"
```